apps/emqx_durable_storage/test/emqx_ds_storage_bitfield_lts_SUITE.erl

```diff
@@ -475,6 +485,12 @@ make_message(PublishedAt, Topic, Payload) when is_binary(Topic) ->
         payload = Payload
     }.
 
+make_topic(T1, T2, T3, T4) ->
+    make_topic([T1, T2, T3, T4]).
```

Comment:
This function looks very specific to a particular testcase. Please remove it and use `make_topic/1` with list argument.